Sources/StreamChatUI/ChatChannelList/ChatChannelListSkeletonView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ChatChannelListSkeletonView.swift
+//  StreamChat
+//
+//  Created by Hugo Bernal on 21/07/22.
+//  Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatChannelListSkeletonView: _View, ThemeProvider, UITableViewDataSource {
+
+    open private(set) lazy var tableView = UITableView().withoutAutoresizingMaskConstraints
+
+    private let numberOfCells = 30
```

Comment:
maybe we should expose this so it is customizable? 🤔

Reply:
Also the number might be too high. Only around 10 are visible, so maybe 15-20 would be more appropriate.

Reply:
Adjusted to 15 and exposed it 👍🏻